core/document/src/color.rs

```diff
@@ -1,5 +1,10 @@
 use serde::{Deserialize, Serialize};
 
+/// Structure that represent a color\
+/// internally alpha is stored as f32 that go from 0.0 (transparent) to 1.0
```

Comment:
Capitalize the start of the new sentence. Replace "go" with "range" and add "(opaque)." after "1.0".